torch/_dynamo/variables/builtin.py

```diff
@@ -176,6 +182,17 @@ def _constant_fold_functions():
     def can_constant_fold_through(self):
         return self.fn in self._constant_fold_functions()
 
+    def is_unfoldable_op(self):
+        unfoldable_ops = {
+           operator.delitem,
+           operator.setitem,
+        } if sys.version_info < (3, 11) else {
```

Comment:
can't you just add operator.call to unfoldable ops manually if `>= 3 11`? Seems less verbose / error prone.

Reply:
will do!

Reply:
ok, done!